lib/code_sponsor_web/templates/template/index.html.eex

```diff
@@ -0,0 +1,51 @@
+<!-- Breadcrumb -->
+<ol class="breadcrumb">
```

Comment:
Can these breadcrumb portions get moved in to view functions that are rendered from a partial? You can do the breadcrumb attributes (link, name) as a function probably and check against the phoenix view (index vs show for instance) being rendered in the function def. Can demonstrate what I'm thinking if needed -- just seems like some duplicated code amongst these.

Reply:
I agree.. I'm just not sure how to do it